src/app.py

```
@@ -31,6 +31,12 @@
 @app.route("/api/<title>", methods=["GET"])
 def get_title(title) -> str:
     # ---
+    # Check for User-Agent header
+    if not request.headers.get('User-Agent'):
+        response_status = "User-Agent missong"
+        logs_db.log_request("/api/<title>", title, response_status, delta)
```

Ruff (0.8.2)

Undefined name `delta`

(F821)